napari/components/dims.py

```diff
@@ -353,28 +300,30 @@ def set_point(self, axis: int, value: Union[int, float]):
         ----------
         axis : int
             Dimension index.
-        value : int or float
+        point : int or float
```

Comment:
You've changed the name in the docstring but not in the signature itself.

Reply:
reverted